### DISCUSSION TITLE
i18n for Vietnamese

### DISCUSSION THREAD

**Discussion by hiepcqn145** [hbstack/.github#58] [General]
Hello,

I understand that I should contribute a Vietnamese translation. However, in my case, and I believe for most Vietnamese people, we would prefer to use English for the blog.

Only the content will be in Vietnamese.

Could you support the creation of an i18n for Vietnamese by copying from English?

**Comment by razonyang** [comment]
No problem, will do in those two days.

**Comment by hiepcqn145** [comment]
Appreciate your support!

**Comment by razonyang** [comment]
Done, please [upgrade all modules](https://hbstack.dev/modules/overview/#upgrade-modules) and setting language code as `vi`.

If you found there is somewhere missed translation, then upgrade one more time, since those modules were just released, you may not get timely upgradation.
